db/reporting_structure.sql

```diff
@@ -15,50 +15,26 @@ SET row_security = off;
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::Parents' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_family_parents VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::ParentingYouth' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_parenting_youth VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::ParentingChildren' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_parenting_children VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::UnaccompaniedMinors' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_unaccompanied_minors VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::IndividualAdults' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_individual_adults VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::NonVeteran' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_non_veteran VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::Family' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_family VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::YouthFamilies' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_youth_families VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::Children' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_children VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'AdultOnlyHouseholdsSubPop::Reporting::MonthlyReports::AdultOnlyHouseholds' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_adult_only_households VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'AdultsWithChildrenSubPop::Reporting::MonthlyReports::AdultsWithChildren' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_adults_with_children VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'ChildOnlyHouseholdsSubPop::Reporting::MonthlyReports::ChildOnlyHouseholds' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_child_only_households VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'ClientsSubPop::Reporting::MonthlyReports::Clients' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_clients VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'NonVeteransSubPop::Reporting::MonthlyReports::NonVeterans' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_non_veterans VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'VeteransSubPop::Reporting::MonthlyReports::Veterans' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_veterans VALUES (NEW.*);
-          
-        ELSE
-          INSERT INTO warehouse_partitioned_monthly_reports_unknown VALUES (NEW.*);
-          END IF;
-          RETURN NULL;
-      END;
+    AS $$
```

Comment:
I'm guessing you should back out these changes as they aren't pertinent to this PR.